README.md

```diff
@@ -1,15 +1,60 @@
-# tket2
+# tket2: The Hardware Agnostic Quantum Compiler
```

Comment:
I feel the hardware agnosticism is not new to TKET2 but is inherited from TKET1, and so in that respect we really are "Version 2 of the TKET compiler" (as you remove), or the sequel of, spawn of, etc. (Now *Hypertket* really is *spawn of* TKET1 for its multithreaded/distributed capabilities, but there...)

I'd say the big emphasis of TKET2 is on composable rewrites, rather than largely-monolithic "passes". That and the hypertket aspects/characteristics. YMMV though....

Reply:
I think the "version 2" is already implicit from the name, and the previous subtitle didn't explain what this lib is really about.

This is a short one-liner, the actual crate content should be explained below. I'm happy to change the subtitle to something better :)